generator/src/main.rs

```diff
@@ -2125,18 +2135,18 @@ fn xr_var_name(raw: &str) -> Ident {
     Ident::new(&raw.to_snake_case(), Span::call_site())
 }
 
-fn xr_arg_ty(member: &Member) -> TokenStream {
+fn xr_arg_ty(api_aliases: &[(String, String)], member: &Member) -> TokenStream {
```

Comment:
@Ralith The cost of resolving aliases was to pass the context through here and to `xr_var_ty` below. These could also become methods of `Parser` and just take `self` instead of only the `api_aliases` context.